ui/lib/ve-token.ts

```diff
@@ -26,23 +26,25 @@ let gasLimits = {
 }
 
 export function useVeNationLock(address: any) {
-  return useContractRead(contractParams, 'locked', {
-    args: [address],
-    watch: true,
-    enabled: !!address,
-    overrides: {
-      gasLimit: gasLimits.locked,
-    },
-  })
+  return useContractRead(
+    {
+      ...contractParams,
+      watch: true,
+      enabled: !!address,
+    }, 
+    'locked', 
+    [address],
+    { gasLimit: gasLimits.locked }
```

Comment:
Is it necessary to use both "locked" and `gasLimits.locked` as parameters?

Reply:
'locked' is the name of the function being called inside the contract.
You're right, there's no need to pass gasLimit since it's a read operation and doesn't consume gas. I've remove it.